src/Drawer/test/DrawerHeaderSpec.tsx

```diff
@@ -1,44 +1,33 @@
 import React from 'react';
 import ReactTestUtils from 'react-dom/test-utils';
 import sinon from 'sinon';
-import { getDOMNode } from '@test/testUtils';
+import { render } from '@testing-library/react';
+import { testStandardProps } from '@test/commonCases';
 
 import Drawer from '../Drawer';
 
 describe('Drawer.Header', () => {
+  testStandardProps(<Drawer.Header />);
   it('Should render a drawer header', () => {
     const title = 'Test';
-    const instance = getDOMNode(<Drawer.Header>{title}</Drawer.Header>);
-    assert.equal(instance.className, 'rs-drawer-header');
-    assert.equal(instance.textContent, 'Test');
+    const { container } = render(<Drawer.Header>{title}</Drawer.Header>);
+    expect(container.firstChild).to.have.class('rs-drawer-header');
+    expect(container.firstChild).to.have.text(title);
   });
 
   it('Should hide close button', () => {
     const title = 'Test';
-    const instance = getDOMNode(<Drawer.Header closeButton={false}>{title}</Drawer.Header>);
-    assert.isNull(instance.querySelector('button'));
+    const { container } = render(<Drawer.Header closeButton={false}>{title}</Drawer.Header>);
+    expect(container.firstChild).to.not.have.class('button');
```

Comment:
The old assertion was checking _a `<button>` does not exists inside of Drawer.Header_, but the new assertion is checking _Drawer.Header does not have `.button` class name_.